src/twin/consent.rs

```diff
@@ -367,7 +391,11 @@ mod tests {
 
         assert_eq!(
             rx_reported_properties.blocking_recv(),
-            Some(json!({"device_update_consent":  {"general_consent":  ["bar", "foo"]}}))
+            Some(json!({
+                "device_update_consent": {
+                    "general_consent": ["bar", "foo"],
+                    "reset_consent_on_fail": false
```

Comment:
@ronny-standtke is it ok from cloud point of view if we additionally report `reset_consent_on_fail`? I assume it is just ignored which is absolutely fine.

Reply:
Will be ignored from cloud side.